en/docs/learn/configuring-account-disabling.md

```diff
@@ -0,0 +1,133 @@
+
+	4.	Select **Supported by Default**.
+
+		<img src="../../assets/img/learn/account-disabled-claim-supported-by-default.png" alt="Account Disabled claim's Suppported by Default option" width="700" style="border:1px solid grey">  
+
+	5. Click **Update**.
+
+## Try out
+
+1.	To create the user account for Larry:
+
+	1.	On the **Main** menu of the Management Console, click **Identity > Users and Roles > Add**.
+
+		<img src="../../assets/img/learn/add-users-and-roles-menu-item.png" alt="Add Users and Roles menu-item" width="200" style="border:1px solid grey">  
+
+	2.	Click **Add New User**.
+
+		<img src="../../assets/img/learn/add-new-user-option.png" alt="Add New User option" width="700" style="border:1px solid grey">  
+
+	3.	Enter the required data as follows.
+
+		<img src="../../assets/img/learn/add-new-user-screen.png" alt="Add New User screen" width="700" style="border:1px solid grey">   
+
+		-	**Domain**: `Primary`
+		-	**Username**: `Larry`
+
+	4.	Click **Finish**. 
+
+2.	To assign login permissions to the user:
+	
+	1.	Click the **View Roles** option of Larry.
+
+		<img src="../../assets/img/learn/view-roles-option.png" alt="View Roles option" width="700" style="border:1px solid grey"> 	 		 		
+
+	2.	Click **Permissions**.
+
+		<img src="../../assets/img/learn/role-permissions-option.png" alt="Role Permissions option" width="700" style="border:1px solid grey"> 	 		 		
+
+	3.	Select **Login** and click **Update**.
+
+		<img src="../../assets/img/learn/login-permission.png" alt="Login permission" width="700" style="border:1px solid grey">  
+
+3.	To disable Larry's user account:
+	
+	1.	Click **User Profile** option of Larry.
+
+		<img src="../../assets/img/learn/user-profile-option.png" alt="User Profile option" width="700" style="border:1px solid grey">   
+
+	2.	Enter an email address to which Larry's account disabling emails will be sent and select the **User Locked** check box.
+
+		<img src="../../assets/img/learn/user-disabled.png" alt="User Disabled option" width="700" style="border:1px solid grey">    		
+
+	3. 	Click **Update**. 
+
+	4.	An email that informs about the account disabling is sent to the given email address.		
+
+		<img src="../../assets/img/learn/account-disabled-email.png" alt="Account Disabled email" width="500" style="border:1px solid grey"> 
+
+	5.	Access the WSO2 Identity Server Dashboard at `https://localhost:9443/dashboard/`.
+
+		<img src="../../assets/img/learn/dashboard-login-screen.png" alt="Sign In form" width="400" style="border:1px solid grey"> 	
+
+	6.	Try logging in with Larry's credentials. Note that an error message appears.
+
+4. To revoke the account disabling:
+
+	1.	Click **User Profile** option of Larry.
+
+	2.	Unselect the **User Disabled** check box.
+
+	3.	Click **Update**
+
+	4.	An email that informs about the account disabling is sent to the given email address.
```

Comment:
"about the disabled account" would be better IMO. WDYT?

Reply:
It should have been the enabled account. :D But, thanks for pointing to this line!